test_src/sljitTest.c

```diff
@@ -11262,6 +11275,12 @@ static void test89(void)
 	FAILED(hbuf[5] != (sljit_s16)0x0201, "test89 case 14 failed\n");
 	FAILED(hbuf[6] != (sljit_s16)0x0201, "test89 case 15 failed\n");
 	FAILED(hbuf[7] != -367, "test89 case 16 failed\n");
+#if IS_64BIT
+	FAILED(hbuf[8] != 0x55aa, "test89 case 17 failed\n");
+	FAILED(buf[5] != hbuf[8], "test89 case 17 (zero extension) failed\n");
```

Comment:
this can be 18, 19,...

Reply:
but using a few extra characters, provide additional information on what part of the behaviour is broken, as well as clarifying that the incorrect value was generated as part of a single "case" code block and therefore aid with troubleshooting.

Reply:
I am ok with the characters, just increase the numbers.